inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -581,5 +581,13 @@ private function set_notice_transient() {
 			time() + 60,
 			MINUTE_IN_SECONDS
 		);
+
+		wp_safe_remote_get(
+			site_url( '/wp-cron.php' ),
+			[
+				'blocking' => false,
+				'timeout'  => 0.01,
+			]
+		);
```

Comment:
What about using this function instead?
https://developer.wordpress.org/reference/functions/spawn_cron/

it's doing the same functionality with additional checks/validations to make sure that we will not affect the performance.

Reply:
Indeed it's much better, I didn't know about that one 👍🏼